src/sys/uio.rs

```diff
@@ -56,6 +59,38 @@ pub fn pread(fd: RawFd, buf: &mut [u8], offset: off_t) -> Result<usize>{
     Errno::result(res).map(|r| r as usize)
 }
 
+// process_vm_{read,write}v currently doesn't use flags, but we define this
```

Comment:
Flags should always be zero in my current version of Linux. I don't think forward compatibility here is a concern because as soon as they're flags we have issues with backwards compatibility on the systems themselves. Let's just leave all this extra boilerplate off for now for the sake of a cleaner API. By the time new flags are added it could be years.